examples/decoding/plot_haxby_anova_svm.py

```diff
@@ -17,8 +17,8 @@
 print('Functional nifti image (4D) is located at: %s' %
       haxby_dataset.func)
```

Comment:
I checked before. It worked not using ```.func[0]```. But, still I can change to be consistent with list. Should I ?